scripts/clangw

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# shellcheck disable=SC2068 disable=SC2006 disable=SC2179
```

Comment:
What code are these disabling warnings for?

Reply:
These warnings occur with the function below and if I fix them, linter is happy but we cannot build on Windows anymore.